ionic2/app/components/poke-filter-popover/poke-filter-popover.component.ts

```diff
@@ -32,7 +21,7 @@ import { FilterService } from '../../services/filter.service';
   ],
 })
 
-export class FilterPopoverComponent {
+export class PokeFilterPopoverComponent {
```

Comment:
"The tag name/selector should mirror the class name." There's what's wrong.

https://angular.io/styleguide